<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Database Encoding</string>
       </property>
       <property name="buddy">
        <cstring>encodingComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Default data for new records</string>
       </property>
       <property name="buddy">
        <cstring>defaultdataComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="defaultdataComboBox">
       <item>
        <property name="text">
         <string>NULL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>''</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Default text editor format</string>
       </property>
       <property name="buddy">
        <cstring>defaultTextComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="defaultTextComboBox">
       <item>
        <property name="text">
         <string>Plain</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Auto</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Default location</string>
       </property>
       <property name="buddy">
        <cstring>locationEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="locationEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="setLocationButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="encodingComboBox">
       <item>
        <property name="text">
         <string notr="true">UTF-8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">UTF-16</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="toolTip">
        <string>Open databases with foreign keys enabled.</string>
       </property>
       <property name="text">
        <string>Foreign keys</string>
       </property>
       <property name="buddy">
        <cstring>foreignKeysCheckBox</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="foreignKeysCheckBox">
       <property name="text">
        <string>enabled</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>encodingComboBox</tabstop>
  <tabstop>defaultdataComboBox</tabstop>
  <tabstop>defaultTextComboBox</tabstop>
  <tabstop>foreignKeysCheckBox</tabstop>
  <tabstop>locationEdit</tabstop>
  <tabstop>setLocationButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>saveSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setLocationButton</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>chooseLocation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveSettings()</slot>
  <slot>defaultDataChanged(int)</slot>
  <slot>encodingChanged(int)</slot>
  <slot>chooseLocation()</slot>
  <slot>defaultTextChanged(int)</slot>
  <slot>foreignkeysStateChanged(int)</slot>
 </slots>
</ui>
